### ISSUE TITLE
Importing a pipeline by its name results in its name as the id

### ISSUE THREAD

**Issue by FreekingDean** [terraform-providers/terraform-provider-heroku#265]
```
Terraform v0.12.24
+ provider.heroku v2.4.0
```

### Affected Resource(s)
`heroku_pipeline`

### Terraform Configuration Files
```hcl
resource "heroku_pipeline"  "some_pipeline" {
  name = "some_pipeline"
}
```

When I import the pipeline from heroku by its name it uses the pipeline `name` as the `ID` instead of the heroku provided UUID.

I replaced my pipeline name with "some_pipeline".

### Expected Behavior
`terraform import heroku_pipeline.main_pipeline some_pipeline` to set the `heroku_pipeline.main_pipeline.id` to eq the heroku supplied UUID.

### Actual Behavior
`terraform import heroku_pipeline.main_pipeline some_pipeline` results in this state:

```hcl
$ terraform state show heroku_pipeline.some_pipeline
resource "heroku_pipeline" "some_pipeline" {
    id   = "some_pipeline"
    name = "some_pipeline"

    owner {
        id   = "[REDACTED]"
        type = "team"
    }
}
```
### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Create a pipeline in heroku
2. Create a terraform file describing the previously created heroku pipeline
3. Run `terraform import heroku_pipeline.<resource_name> <your heroku pipeline *name*>`

### Important Factoids
When importing I used the heroku pipeline name instead of the ID. I'm guessing this was the cause.

Importing it by the UUID does result in the proper state.

**Comment by bernerdschaefer** [comment]
Pipelines currently only support import by ID: https://www.terraform.io/docs/providers/heroku/r/pipeline.html#import.

Here's the relevant import code:

https://github.com/terraform-providers/terraform-provider-heroku/blob/434ce64ef0cab928a758e2415a677ff8fcca3c22/heroku/resource_heroku_pipeline.go#L60-L71

I do think that if there was a call to `d.SetId(p.ID)` before line 68, then importing pipelines would behave as you expected when providing a name instead of the ID for import.

**Comment by davidji99** [comment]
Should be a quick fix. I'll fire up a PR.

**Comment by mars** [comment]
Fix released in [v2.4.1](https://github.com/terraform-providers/terraform-provider-heroku/releases/tag/v2.4.1) 🏁
